README.md

```diff
@@ -165,19 +165,21 @@ hasn't been ruled out, but neither is it being actively pursued today.
 Related project comparison: runc/binctr
 ----------------------------------------
 
-[runc](https://github.com/opencontainers/runc) is similar to
-[systemd nspawn](https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html)
-in that it is tooling intended to be invoked by root.  There is an
-effort to have runc optionally use
-[user namespaces](https://github.com/opencontainers/runc/issues/38),
-but no plans for any setuid support.
+[runC](https://github.com/opencontainers/runc) is currently working on
+supporting [rootless containers](https://github.com/opencontainers/runc/pull/774),
+without needing `setuid` or any other privileges during installation of
+runC (using unprivileged user namespaces rather than `setuid`),
+creation, and management of containers. However, the standard mode of
+using runC is similar to [systemd nspawn](https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html)
+in that it is tooling intended to be invoked by root.
 
 The bubblewrap authors believe that runc and systemd-nspawn are not
-designed to be made setuid and are distant from supporting such a
-mode.
+designed to be made setuid, and are distant from supporting such a mode.
+However with rootless containers, runC will be able to fulfil certain
+requirements that bubblewrap cannot.
```

Comment:
Specifically, the requirement for rootless containers is that you have a user on a system that doesn't have any admin privileges (and the administrator won't install things without you going through several weeks of bureaucracy). This is something that happens quite a lot with academic computing clusters. You need to use a container to package your analysis scripts, but you can't use a container and you can't get the admin to install the libraries you need.